src/index.js

```diff
@@ -77,20 +80,35 @@ const handleHideAll = (target, groupTarget, toggleClass) => {
 
 const setupToggle = () => {
 
+    /**
+     * If accordion, only display first section on initialisation
+     */
+
+    $('[data-toggle-accordion]')
+        .forEach(accordion => {
+            const toggleClass = accordion.getAttribute('data-toggle-class') || 'is-hidden';
+            const sections = [...accordion.querySelectorAll('[data-toggle-name]')];
+
+            sections.splice(0, 1);
+            sections
+                .forEach(toggles(toggleClass).hide);
+        });
+
     /**
      * Bind the toggle element click events
      */
+
     $('[data-toggle-target]')
         .forEach(toggle => {
             toggle.addEventListener('click', e => {
                 e.preventDefault();
 
                 const target = toggle.getAttribute('data-toggle-target');
                 const toggleClass = toggle.getAttribute('data-toggle-class') || 'is-hidden';
-                const groupTarget = toggle.getAttribute('data-toggle-group-target');
+                const accordion = toggle.closest('[data-toggle-accordion]');
```

Comment:
Support for `closest` is a little flakey, might be worth noting in the readme that this may need to be polyfilled?